src/opm/parser/eclipse/EclipseState/Schedule/Group/Group.cpp

```diff
@@ -55,8 +55,7 @@ namespace Opm {
         m_injection( timeMap ),
         m_production( timeMap ),
         m_wells( timeMap, {} ),
-        m_isProductionGroup( timeMap, false),
-        m_isInjectionGroup( timeMap, false),
+        group_type(timeMap, GroupType::NONE),
```

Comment:
Initialising `group_type` to `NONE` and then never, as far as I could tell, clearing that bit means that
```C++
(group_type.get(tstep) & GroupType::NONE) == GroupType::NONE
```
is always true.  If that's a problem, we need to ensure that we never ask that question.